glotaran/project/result.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+from math import nan
```

Comment:
Why use another import when we already have `numpy` and can use `np.nan`?

Reply:
Changed PR to avoid additional import